index.ts

```diff
@@ -156,6 +171,9 @@ async function importDocument(folderEntryId: number | undefined, sampleProjectFi
   return importedEntryId;
 }
 
+/**
+ * Sets fields on the entry specified by the given entry Id.
```

Comment:
Maybe want to be more specific here like 
`Sets a string field on the entry specified by the given entry Id.`

Reply:
Fixed.